tests/unit/cluster/cluster-reliable-meet.tcl

```diff
@@ -0,0 +1,79 @@
+# make sure the test infra won't use SELECT
+set old_singledb $::singledb
+set ::singledb 1
+
+tags {tls:skip external:skip cluster} {
+
+set base_conf [list cluster-enabled yes]
+start_multiple_servers 2 [list overrides $base_conf] {
+
+test "Cluster nodes are reachable" {
```

Comment:
Some tests have this indentation since we wanted to preserve git history, when we ported them from the old to new framework. For new tests, they should ideally be indented. Maybe we should just format them at this point, since we are already losing so much history.

Reply:
Thanks! I was wonder why the indent was off and thought it might be something to do with tcl syntax. Will fix it.